cazomevolve/utilities/parsers/get_dbcan_parser.py

```diff
@@ -104,6 +104,14 @@ def build_parser(
         help="enable/disable deletion of exisiting files",
     )
 
+    parser.add_argument(
+        "--toolcount",

```

Comment:
Change arg name to `tool_count`
```suggestion
        "--tool_count",

```

Reply:
I have changed the argument to tool_count throughout the parser and python script, and changed the version number from 0.1.7.3 to 0.1.7.4 in the __init__.py